docs/reference/filebeat/filebeat-input-filestream.md

```diff
@@ -325,53 +325,88 @@ The setting relies on the modification time of the file to determine if a file i
 
 To remove the state of previously harvested files from the registry file, use the `clean_inactive` configuration option.
 
-## Take over [filebeat-input-filestream-take-over]
+#### `take_over` [filebeat-input-filestream-take-over]
+
+```{applies_to}
+stack: beta
+```
+
+::::{warning}
+The take over mode is still in beta, however, it should be generally safe to use.
+::::
+
 When `take_over` is enabled, this `filestream` input will take over
 states from the [`log`](/reference/filebeat/filebeat-input-log.md) input
 or other `filestream` inputs. Only states of files being actively
 harvested by this input are taken over.
 
-To take over files from a `log` input, simply set `take_over.enabled: true`.
+The syntax for enabling take over mode varies by version:
+
+* {applies_to}`stack: beta 9.0.0` Use `take_over: true`.
+* {applies_to}`stack: beta 9.1.0` Use `take_over.enabled: true`.
+
+:::{note}
+While `take_over: true` is still supported to migrate state from the `log` input to
+`filestream`, we plan to remove support in a future version so you should use
+the new syntax if possible.
+:::
+
+##### `log` input
+
+To take over files from a `log` input, enable take over mode
+and make sure the files you want this input to take over match the configured globs in `paths`.
+
+::::{tab-set}
+:::{tab-item} 9.1.0
+```yaml
+take_over:
+  enabled: true
+```
+:::
+:::{tab-item} 9.0.0
+```yaml
+take_over: true
+```
+:::
+::::
+
 
-To take over states from other `filestream` inputs, set
-`take_over.enabled: true` and set `take_over.from_ids` to a list of
-existing `filestream` IDs you want to migrate files from.
 
-On both cases make sure the files you want this input to take over
-match the configured globs in `paths`.
+##### Other `filestream` inputs
+
+```{applies_to}
+stack: beta 9.1.0
+```
+
+To take over states from other `filestream` inputs, enable take over mode,
+set `take_over.from_ids` to a list of existing `filestream` IDs you want to migrate files from,
+and make sure the files you want this input to take over match the configured globs in `paths`.
 
 When `take_over.from_ids` is set, files are not taken over from `log`
 inputs. The migration is limited to `filestream` inputs only.
 
 ```yaml
 take_over:
   enabled: true
-  from_ids: ["foo", "bar"] # omit to take over from the log input
+  from_ids: ["foo", "bar"]
 ```
```

Comment:
Based on the example in the code snippet, I _think_ this is only possible starting in 9.1.0. Is that correct?